worksheet_B/Mandelbrot.cpp

```diff
@@ -3,6 +3,125 @@
 
 #include "stdafx.h"
 
+
+//hsv to rgb converter https://www.programmingalgorithms.com/algorithm/hsv-to-rgb?lang=C%2B%2B
```

Comment:
Quite a lot of code here -- might have been better to take it out into a separate file